web/pgadmin/tools/schema_diff/static/js/SchemaDiffModule.js

```diff
@@ -79,7 +80,7 @@ export default class SchemaDiff {
       'pgadmin:tool:show',
       `${BROWSER_PANELS.SCHEMA_DIFF_TOOL}_${trans_id}`,
       baseUrl,
-      {},
+      {sql_id: sqlId},
```

Comment:
the `sqlId` is not really SQL Id. You can think some other var name.

Reply:
Done.